.gitignore

```diff
@@ -94,7 +95,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
-
+.idea/
```

Comment:
This is for IntelliJ IDEA as I understand it, so let's add a comment about that.

As you can see, all other sections are formatted, i.e. separated by newline and with a comment, like
```
# Rope project settings
.ropeproject
```

Reply:
ok i will do it